### PR TITLE
refactor: remove the unused StakeChunkOnlyAction

### PR DESCRIPTION
**Pull request by matklad**
closes #CPR-83

We figured out that we don't actually need a separate stake action for
chunk-only producers.

### CODE REVIEW

**Comment by matklad** [comment]
@mm-near is this what you had in mind for this issue? Regarding veryfying that it works, it seems that nothing in our system, not even tests, were producing that stacking action (well, I suppose some code might be sending borsh bytes bypassing the struct, but that seems unlikely).

File: runtime/runtime/src/actions.rs [hunk 1]
```diff
@@ -308,7 +307,7 @@ pub(crate) fn action_stake(
             stake.public_key.clone(),
             stake.stake,
             #[cfg(feature = "protocol_feature_chunk_only_producers")]
-            is_chunk_only,
+            false,
```
Comment (mm-near) [inline runtime/runtime/src/actions.rs:310]: do we still need this parameter ? we probably can also remove ValidatorStakeV2

**Review by mm-near — APPROVED** [review]

File: runtime/runtime/src/actions.rs (hunk 1 above)
Comment (matklad) [reply runtime/runtime/src/actions.rs:310]: Yeah, thought the same: https://nearinc.atlassian.net/browse/CPR-83?focusedCommentId=15564

I guess you suggestion means that we don't need the concept of chunk-only stake, so I'll remove that in a follow-up.

Comment (matklad) [reply runtime/runtime/src/actions.rs:310]: https://github.com/near/nearcore/pull/7085

**Pull request merged**